mobile/src/Student/components/Answers/Answer/index.js

```diff
@@ -1,129 +1,200 @@
-import React from 'react'
-import { StyleSheet, Text, View, Image } from 'react-native'
-import { scale, verticalScale } from 'react-native-size-matters'
+import React, { useState, useRef } from "react";
+import { StyleSheet, Text, View, Image, Animated } from "react-native";
+import { scale } from "react-native-size-matters";
+import LinearGradient from "react-native-linear-gradient";
+import * as Progress from "react-native-progress";
 import { fontFamilies, fonts } from '../../../../utils/theme'
-import LinearGradient from 'react-native-linear-gradient'
-import * as Progress from 'react-native-progress'
 
-const Answer = ({ answer, isPopular, percentage, isAnswer }) => {
-    const popularIconVisible = isPopular || false
-    const percentageVisible = !popularIconVisible && percentage !== undefined
-    const percentageTitle = (percentage) => {
-        return percentage + '%'
-    }
-    return (
-        <View style={[
-            styles.container,
-            {
-                backgroundColor: isAnswer ? '#6515C9' : 'white'
-            }
-        ]}>
-            <View style={styles.answerContainer}>
-                <Text
-                    style={[
-                        styles.answer,
-                        {
-                            color: isAnswer ? 'white' : '#384466'
-                        }
-                    ]}
-                >
-                    {answer}
-                </Text>
-                {
-                    (percentage || false) &&
-                    <Progress.Bar
-                        style={styles.progressBar}
-                        progress={percentage / 100}
-                        color={'#3AC449'}
-                        unfilledColor={'rgba(255,255,255,0.8)'}
-                    />
-                }
-            </View>
+const Answer = ({
+  answer,
+  isPopular,
+  percentage,
+  isAnswer,
+  waitTime,
+  totalAns,
```

Comment:
I believe it's shorten for `totalAnswers`. If so, would be nice not to shorten so help with readability